src/client.rs

```diff
@@ -17,51 +11,8 @@ use crate::value::{FromMemcacheValueExt, ToMemcacheValue};
 
 pub type Stats = HashMap<String, String>;
 
-pub trait Connectable {
-    fn get_urls(self) -> Vec<String>;
-}
-
-impl Connectable for String {
-    fn get_urls(self) -> Vec<String> {
-        return vec![self];
-    }
-}
-
-impl Connectable for Vec<String> {
-    fn get_urls(self) -> Vec<String> {
-        return self;
-    }
-}
-
-impl Connectable for &str {
-    fn get_urls(self) -> Vec<String> {
-        return vec![self.to_string()];
-    }
-}
-
-impl Connectable for Vec<&str> {
-    fn get_urls(self) -> Vec<String> {
-        let mut urls = vec![];
-        for url in self {
-            urls.push(url.to_string());
-        }
-        return urls;
-    }
-}
-
 #[derive(Clone)]
```

Comment:
Not possible to derive debug implementation?

Reply:
possible, derived :+1: